tests/parsers/beckman_pharmspec/beckman_pharmspec_parser_test.py

```diff
@@ -22,129 +22,223 @@
 
 
 @pytest.fixture()
-def test_file() -> Path:
+def test_file_1() -> Path:
```

Comment:
Can we replace these with a list of test file names, and a couple helpers to get paths from them? Similar to other test files.

See `vi_cell_blu_parser_test.py` as one example.